qiita_ware/ebi.py

```diff
@@ -272,8 +271,8 @@ def generate_study_xml(self):
 
         Returns
         -------
-        str
-            string with study XML values
+        Element
+            Object with study XML values
```

Comment:
Dumb question.  What exactly is `Element`?  Is it an xml element corresponding to `xml.etree`?  It would be nice to include the object type in the docs